Gemfile.rails-5.0.0

```diff
@@ -0,0 +1,6 @@
+source "https://rubygems.org"
+
+gem 'rails', '5.0.0'
```

Comment:
it sounds like further improvement, initial goal was to have travis matrix for 4.2.6 and 5.0.0, sure many approaches possible like 4.2-stable, 5.0-stable, last-master etc -- feel free to add changes, or using branch as an example for improvements/alternatives